modules/dns_nsupdate/dns_nsupdate_gss_main.rb

```diff
@@ -26,4 +26,4 @@ def nsupdate cmd
       super
     end
   end
-end
+end
```

Comment:
Nitpick, if you can configure your IDE/editor to keep end of file newlines, it'd be appreciated.